Makefile

```diff
@@ -56,7 +56,7 @@ local-run:
 	--hostname 127.0.0.1 \
 	--deployment_mode 0 \
 	--keys_file ../docker.local/keys_config/b0bnode1_keys.txt  \
-	--files_dir ./data/blobber/files \
+	--mount_point ./data/blobber/files \
```

Comment:
please keep it with `--files_dir`, if not. it will break all helm charts.

Reply:
or please update it on dockerfile and helm charts too

Reply:
I think its better to update in cli and helm charts. For files storage, It must be mountpoint. So the flag name should be appropriate as well.